app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -198,6 +199,12 @@ class Settings private constructor(
         true
     )
 
+    val shouldUseFixedToolbar: Boolean
+        get() {
+            val accessibilityManager = appContext.getSystemService(Context.ACCESSIBILITY_SERVICE) as? AccessibilityManager
```

Comment:
What does fixed toolbar mean, and how does it relate to accessibility?  Would you add a short (1 or 2 lines) comment here explaining?